app/pom.xml

```diff
@@ -43,6 +43,7 @@
          <plugin>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-maven-plugin</artifactId>
+            <version>2.6.3</version>
```

Comment:
Shouldn't the version be dictated by the watcher?

Reply:
The openmrs watcher is pulled in as default type jar instead of as pom which I think required @enyachoke to add this change. @enyachoke could you clarify why you had to add this in?